experiments.md

```diff
@@ -81,38 +83,38 @@ Experiment results for each method will be saved under `results/<method>`.
 
 ### Postgres
 
-- Setup Postgres (scale `SCALE`, partition 100):
+- Setup Postgres (scale `SCALE`, partition `PARTITION`):
 ```bash
 export QUERY_DIR=./resources/tpc-h/queries
-export POSTGRES_DIR=./tmp/postgres/scale=${SCALE}/partition=100
-./baselines/postgres/experiment-setup.sh ${DATA_DIR} ${QUERY_DIR} ${POSTGRES_DIR} ${SCALE} 100
+export POSTGRES_DIR=./tmp/postgres/scale=${SCALE}/partition=${PARTITION}
+./baselines/postgres/experiment-setup.sh ${DATA_DIR} ${QUERY_DIR} ${POSTGRES_DIR} ${SCALE} ${PARTITION}
```

Comment:
Since this is not in a docker, could you as psql as a prerequisite? Otherwise we'd need one more

Reply:
The script actually runs `psql` inside a docker container. So, don't need to add the dependency.